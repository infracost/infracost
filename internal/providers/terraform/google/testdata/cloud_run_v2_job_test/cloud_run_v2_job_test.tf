provider "google" {
  credentials = "{\"type\":\"service_account\"}"
  project     = "my-project"
}

resource "google_cloud_run_v2_job" "my_job" {
  name     = "cloudrun-v2-job-test"
  location = "europe-west3"
  template {
    template {
      containers {
        image = "us-docker.pkg.dev/cloudrun/container/hello"
      }
    }
    task_count = 1
  }
}

resource "google_cloud_run_v2_job" "no_usage" {
  name     = "cloudrun-v2-job-test"
  location = "europe-west3"
  template {
    template {
      containers {
        image = "us-docker.pkg.dev/cloudrun/container/hello"
      }
    }
    task_count = 1
  }
}
