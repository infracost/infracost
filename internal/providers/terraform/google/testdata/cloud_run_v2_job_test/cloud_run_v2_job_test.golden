
 Name                                       Monthly Qty  Unit                      Monthly Cost   
                                                                                                  
 google_cloud_run_v2_job.my_job                                                                   
 ├─ CPU allocation time (tier 2)             30,000,000  vCPU-seconds                   $648.00   
 └─ Memory allocation time (tier 2)          15,000,000  GiB-seconds                     $36.00   
                                                                                                  
 google_cloud_run_v2_job.no_usage                                                                 
 ├─ CPU allocation time (tier 2)     Monthly cost depends on usage: $0.0000216 per vCPU-seconds   
 └─ Memory allocation time (tier 2)  Monthly cost depends on usage: $0.0000024 per GiB-seconds    
                                                                                                  
 OVERALL TOTAL                                                                         $684.00 

*Usage costs can be estimated by updating Infracost Cloud settings, see docs for other options.

──────────────────────────────────
2 cloud resources were detected:
∙ 2 were estimated

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━┓
┃ Project                                            ┃ Baseline cost ┃ Usage cost* ┃ Total cost ┃
┣━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╋━━━━━━━━━━━━━━━╋━━━━━━━━━━━━━╋━━━━━━━━━━━━┫
┃ main                                               ┃          $684 ┃           - ┃       $684 ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┻━━━━━━━━━━━━━━━┻━━━━━━━━━━━━━┻━━━━━━━━━━━━┛